### PR TITLE
feat: remove inline c

### PR DESCRIPTION
**Pull request by ask747**

### CODE REVIEW

File: .idea/misc.xml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (etceterum) [inline .idea/misc.xml]: I'd delete all `.idea` stuff from version control and (optionally) put `.idea` into gitignore

File: .travis.yml [hunk 2]
```diff
@@ -1,17 +0,0 @@
-language: ruby
```
Comment (etceterum) [inline .travis.yml]: I'd keep `.travis.yml` if it doesn't break anything for you. When you fork a gem I think it is beneficial to keep your version as close as possible to the original so it is both easier for you to upgrade your fork if the original gem gets an update as well as for you to contribute to the original gem by creating a PR against it if you choose to do so

File: fuzzy_tools.gemspec [hunk 3]
```diff
@@ -17,7 +17,7 @@ Gem::Specification.new do |s|
   s.executables   = `git ls-files -- bin/*`.split("\n").map{ |f| File.basename(f) }
   s.require_paths = ["lib"]
 
-  s.add_dependency 'RubyInline'
```
Comment (etceterum) [inline fuzzy_tools.gemspec:20]: I'd delete the extra blank lines

File: lib/fuzzy_tools/helpers.rb [hunk 4]
```diff
@@ -30,8 +30,6 @@ def ngrams(str, n)
       (0..str.length - n).map { |i| str[i,n] }
     end
 
-    if RUBY_DESCRIPTION !~ /^ruby/ # rbx, jruby
-
       SOUNDEX_LETTERS_TO_CODES = {
```
Comment (etceterum) [inline lib/fuzzy_tools/helpers.rb]: Please align the lines that used to be inside `if` with the rest of the code (basically, unindent 1 level)

File: lib/fuzzy_tools/weighted_document_tokens.rb [hunk 5]
```diff
@@ -9,7 +9,7 @@ def initialize(tokens, options)
       set_token_weights(tokens, &weight_function)
     end
 
-    if RUBY_DESCRIPTION !~ /^ruby/
+
 
       # Rubinius and JRuby
```
Comment (etceterum) [inline lib/fuzzy_tools/weighted_document_tokens.rb]: Please unindent the code that used to be inside the `if` block

File: spec/enumerable_spec.rb [hunk 6]
```diff
@@ -24,27 +25,32 @@
     end
 
     context "passes :tokenizer through to the index" do
-      before(:each) { @letter_count_tokenizer = lambda { |str| str.size.to_s } }
+
+      before(:each) {
```
Comment (etceterum) [inline spec/enumerable_spec.rb]: The G2 convention for blocks is to only use the curlies to denote a single-line block; For multiline, we use `do` .. `end`

File: spec/enumerable_spec.rb [hunk 7]
```diff
@@ -54,18 +60,21 @@
     end
 
     context "passes :tokenizer through to the index" do
-      before(:each) { @letter_count_tokenizer = lambda { |str| str.size.to_s } }
+      before(:each) {
```
Comment (etceterum) [inline spec/enumerable_spec.rb]: Same comment for multiline block as above

**Review by etceterum — CHANGES_REQUESTED** [review]

File: .idea/misc.xml (hunk 1 above)
Comment (ask747) [reply .idea/misc.xml]: Done ))

File: .travis.yml (hunk 2 above)
Comment (ask747) [reply .travis.yml]: Done

**Review by etceterum — APPROVED** [review]
LGTM

**Pull request merged**